tavern/config.go

```diff
@@ -16,6 +16,9 @@ import (
 )
 
 var (
+	// Port to run on
+	EnvBind = EnvString{"TAVERN_BIND", "127.0.0.1:8080"}
```

Comment:
Let's name this `EnvHTTPListenAddr = EnvString{"HTTP_LISTEN_ADDR", "0.0.0.0:80"}`, also please keep the current defaults.